### ISSUE TITLE
车牌检测，loss值1万多

### ISSUE THREAD

**Issue by buptys2016** [lsr12345/tensorflow2.x-yolov3#1]
您好，请问一下，我做车牌检测，也就是只有一类，但是loss值一直一万多，不下降，这是什么原因。

**Comment by lsr12345** [comment]
maybe you can change the learning rate scheduler or update code from this repositories, also this repositories without pretrained weights, so need more time to train in your dataset
